### ISSUE TITLE
Parameters for init script install_gpu_driver.sh to install diff GPU drivers

### ISSUE THREAD

**Issue by NvTimLiu** [GoogleCloudDataproc/initialization-actions#1012]
For Rocky/Ubuntu GCP images, the GPU driver version in install_gpu_driver.sh is hardcode to 460 as below:

ubuntu image : https://github.com/GoogleCloudDataproc/initialization-actions/blob/master/gpu/install_gpu_driver.sh#L207

Rocky image : https://github.com/GoogleCloudDataproc/initialization-actions/blob/master/gpu/install_gpu_driver.sh#L212

Can it be parameterized, then user can overwrite the GPU driver to a customized version 450/495, just like what Debian do? 

```
# Parameters for NVIDIA-provided Debian GPU driver
readonly DEFAULT_NVIDIA_DEBIAN_GPU_DRIVER_VERSION='460.73.01'
```

**Comment by jayadeep-jayaraman** [comment]
@nvliyuan - can you pls take a look into this ?

**Comment by nvliyuan** [comment]
will draft a new pr to address this issue after #1018 merged

**Comment by cjac** [comment]
> For Rocky/Ubuntu GCP images, the GPU driver version in install_gpu_driver.sh is hardcode to 460 as below:
> 
> ubuntu image : https://github.com/GoogleCloudDataproc/initialization-actions/blob/master/gpu/install_gpu_driver.sh#L207
> 
> Rocky image : https://github.com/GoogleCloudDataproc/initialization-actions/blob/master/gpu/install_gpu_driver.sh#L212
> 
> Can it be parameterized, then user can overwrite the GPU driver to a customized version 450/495, just like what Debian do?
> 
> ```
> # Parameters for NVIDIA-provided Debian GPU driver
> readonly DEFAULT_NVIDIA_DEBIAN_GPU_DRIVER_VERSION='460.73.01'
> ```

readonly NVIDIA_DEBIAN_GPU_DRIVER_VERSION=$(get_metadata_attribute 'gpu-driver-version' ${DEFAULT_NVIDIA_DEBIAN_GPU_DRIVER_VERSION})

Yes, the gpu-driver-version metadata property will select the GPU driver version.  Use with:

`--metadata gpu-driver-version=510.47.03`

**Comment by cjac** [comment]
> For Rocky/Ubuntu GCP images, the GPU driver version in install_gpu_driver.sh is hardcode to 460 as below:
> 
> ubuntu image : https://github.com/GoogleCloudDataproc/initialization-actions/blob/master/gpu/install_gpu_driver.sh#L207
> 
> Rocky image : https://github.com/GoogleCloudDataproc/initialization-actions/blob/master/gpu/install_gpu_driver.sh#L212
> 
> Can it be parameterized, then user can overwrite the GPU driver to a customized version 450/495, just like what Debian do?
> 
> ```
> # Parameters for NVIDIA-provided Debian GPU driver
> readonly DEFAULT_NVIDIA_DEBIAN_GPU_DRIVER_VERSION='460.73.01'
> ```

readonly NVIDIA_DEBIAN_GPU_DRIVER_VERSION=$(get_metadata_attribute 'gpu-driver-version' ${DEFAULT_NVIDIA_DEBIAN_GPU_DRIVER_VERSION})

Yes, the gpu-driver-version metadata property will select the GPU driver version.  Use with:

`--metadata gpu-driver-version=510.47.03`

**Comment by cjac** [comment]
Fixed in PR #1023
